<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>540</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="main_page">
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>440</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>type to search...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
      </font>
     </property>
     <property name="text">
      <string>INVENTORY</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>500</width>
       <height>300</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>484</width>
        <height>330</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_1">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_3">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_6">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_5">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_4">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_7">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_8">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_9">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>80</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Search
(wip)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="settingsButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Settings
(wip)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="powerButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Power
(wip)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>460</y>
       <width>371</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Outfit Selection (wip)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newItemButton">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>460</y>
       <width>111</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Item</string>
     </property>
    </widget>
    <widget class="QPushButton" name="debugPrintButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>DEBUG
Print Closet</string>
     </property>
    </widget>
    <widget class="QPushButton" name="debugImageButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>330</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>DEBUG
Take Image</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="register_page">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
      </font>
     </property>
     <property name="text">
      <string>REGISTER NEW ITEM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>100</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RFID tag undetected... (wip)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="idInput">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>140</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameInput">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>180</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="tagInput">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>320</y>
       <width>301</width>
       <height>70</height>
      </rect>
     </property>
     <property name="plainText">
      <string>tag 1
tag 2
tag 3
tag 4</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>290</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>prexisting tag 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>prexisting tag 2</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>290</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Add tag</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="imageInput">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>220</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="goBackButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>430</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Go Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmRegisterButton">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>430</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>140</y>
       <width>57</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>180</y>
       <width>57</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>220</y>
       <width>57</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Image:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>220</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Take Image</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="edit_page">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Edit Item</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>280</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>prexisting tag 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>prexisting tag 2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>210</y>
       <width>57</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Image:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>210</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Take Image</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="idInput_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>130</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="tagInput_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>310</y>
       <width>301</width>
       <height>70</height>
      </rect>
     </property>
     <property name="plainText">
      <string>tag 1
tag 2
tag 3
tag 4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>280</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Add tag</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>170</y>
       <width>57</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="goBackButton_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>420</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Go Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>130</y>
       <width>57</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmRegisterButton_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>420</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameInput_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>170</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="imageInput_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>210</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
